### PR TITLE
Link to article instead of dedicated comment page in recent-comment sidebar

### PR DESCRIPTION
**Pull request by patbl**
Fixes tog22/eaforum#75.

### CODE REVIEW

**Comment by peterhurford** [comment]
LGTM :+1: :shipit:

File: r2/r2/models/link.py [hunk 1]
```diff
@@ -408,9 +408,6 @@ def slug():
             res = "/%s" % p
         return res
 
-    def make_permalink_slow(self):
-        return self.make_permalink(self.subreddit_slow)
-
```
Comment (moggyboy) [inline r2/r2/models/link.py]: It looks like you deleted `Link#comment`, assuming that `Comment#comment` was a duplicate method. There are many class definitions in that file.

Comment (patbl) [inline r2/r2/models/link.py]: Oh golly, you're right. I'm accustomed to one class per file.

Comment (moggyboy) [inline r2/r2/models/link.py]: So are most people who work in sane, well-organized, well-documented codebases :-S

Comment (moggyboy) [inline r2/r2/models/link.py]: For the record, I stared blankly at the screen for a while before that occurred to me, and I've been writing software professionally over 20 years, so don't feel bad.

**Pull request merged**